src/media/css/elements--nav.styl

```diff
@@ -105,16 +111,16 @@ body {
 mkt-nav {
     font-size: 18px;
     height: 100%;
-    left: 0;
-    opacity: 0;
+    left: -1 * $nav--width;
```

Comment:
When I first pulled in Chuck's changes I also thought I'd switch it to always use `translate3d` but my initial attempt ended up causing funkiness with the scroll bar (if it was visible it would animate faster than the content) and this doesn't so I went with it.